superset-frontend/src/views/CRUD/dashboard/DashboardCard.tsx

```diff
@@ -139,7 +141,7 @@ function DashboardCard({
     <CardStyles
       onClick={() => {
         if (!bulkSelectEnabled) {
-          window.location.href = dashboard.url;
+          history.push(dashboard.url);
```

Comment:
Let's do an E2E test to go into and out of a dashboard, to make sure we can get hither and yon.

Reply:
Would be good to have one that hits the dashboard url directly and one that navigates to it via the list view link